test/ao/sparsity/test_parametrization.py

```diff
@@ -0,0 +1,139 @@
+        assert hasattr(model.seq[1], 'parametrizations')
+        assert parametrize.is_parametrized(model.linear, 'weight')
+
+    def test_state_dict_preserved(self):
+        model_save = ModelUnderTest(bias=False)
+
+        parametrize.register_parametrization(model_save.linear, 'weight',
+                                             utils.FakeSparsity(model_save.linear.mask))
+        parametrize.register_parametrization(model_save.seq[0], 'weight',
+                                             utils.FakeSparsity(model_save.seq[0].mask))
+        parametrize.register_parametrization(model_save.seq[1], 'weight',
+                                             utils.FakeSparsity(model_save.seq[1].mask))
+        state_dict = model_save.state_dict()
+
+        model_load = ModelUnderTest(bias=False)
+        parametrize.register_parametrization(model_load.linear, 'weight',
+                                             utils.FakeSparsity(torch.zeros(model_load.linear.weight.shape)))
+        parametrize.register_parametrization(model_load.seq[0], 'weight',
+                                             utils.FakeSparsity(torch.zeros(model_load.seq[0].weight.shape)))
+        parametrize.register_parametrization(model_load.seq[1], 'weight',
+                                             utils.FakeSparsity(torch.zeros(model_load.seq[1].weight.shape)))
+        model_load.load_state_dict(state_dict)
+
+        # Check the parametrizations are preserved
+        assert hasattr(model_load.linear, 'parametrizations')
+        assert parametrize.is_parametrized(model_load.linear, 'weight')
+        assert hasattr(model_load.seq[0], 'parametrizations')
+        assert parametrize.is_parametrized(model_load.linear, 'weight')
+        assert hasattr(model_load.seq[1], 'parametrizations')
+        assert parametrize.is_parametrized(model_load.linear, 'weight')
+
+        # Check the weigths are preserved
+        self.assertEqual(model_save.linear.parametrizations['weight'].original,
+                         model_load.linear.parametrizations['weight'].original)
+        self.assertEqual(model_save.seq[0].parametrizations['weight'].original,
+                         model_load.seq[0].parametrizations['weight'].original)
+        self.assertEqual(model_save.seq[1].parametrizations['weight'].original,
+                         model_load.seq[1].parametrizations['weight'].original)
+
+        # Check the masks are preserved
+        self.assertEqual(model_save.linear.parametrizations['weight'][0].mask,
+                         model_load.linear.parametrizations['weight'][0].mask)
+        self.assertEqual(model_save.seq[0].parametrizations['weight'][0].mask,
+                         model_load.seq[0].parametrizations['weight'][0].mask)
+        self.assertEqual(model_save.seq[1].parametrizations['weight'][0].mask,
+                         model_load.seq[1].parametrizations['weight'][0].mask)
+
+    def test_jit_trace(self):
```

Comment:
Can you also add a test to check if parameterized models are scriptable?

Reply:
The parametrizations are not scriptable

Reply:
Please mark a TODO to add tests for symbolic tracing also. We need paramaterization to compose with these APIs.